src/xcode/ENA/ENA/Source/Utils/__tests__/DidSetPublishedTests.swift

```diff
@@ -0,0 +1,48 @@
+////
+// 🦠 Corona-Warn-App
+//
+
+import XCTest
+@testable import ENA
+
+class DidSetPublishedTests: XCTestCase {
+
+    func testExample() throws {
+		let publishingStruct = PublishingStruct()
+
+		let expectedValues = [false, false, true, true, false]
+
+		let expectation = expectation(description: "Received value")
+		expectation.expectedFulfillmentCount = expectedValues.count
+
+		var receivedValues = [Bool]()
+		let subscription = publishingStruct.$publishingBool
+			.sink {
+				// Check that the value is already set on the publishing struct and we receive it afterwards
+				XCTAssertEqual($0, publishingStruct.publishingBool)
+				receivedValues.append($0)
+
+				expectation.fulfill()
+			}
+
+		publishingStruct.publishingBool = false
+		publishingStruct.publishingBool = true
+		publishingStruct.publishingBool = true
+		publishingStruct.publishingBool = false
+
+		waitForExpectations(timeout: .short)
+
+		XCTAssertEqual(receivedValues, expectedValues)
+
+		subscription.cancel()
```

Comment:
Just out of interest, wouldn't this cancel automatically as the `subscription` is only scoped in this function?

Reply:
I think that's true, it's just an elegant way for me to silence the `Initialization of immutable value 'subscription' was never used; consider replacing with assignment to '_' or removing it` warning